tests/core/full_node/test_full_node.py

```diff
@@ -613,7 +613,7 @@ async def test_request_sub_blocks(self, wallet_nodes):
         blocks = await full_node_1.get_all_full_blocks()
 
         blocks = bt.get_consecutive_blocks(
-            30,
+            33,
```

Comment:
As someone new to this codebase, it's really hard to know the significance of this number. I think there should be a comment here, explaining why 33 was chosen here, maybe even with a reference to the limit that I think you mentioned earlier.